packages/components/src/components/bal-slider/bal-slider-item/bal-slider-item.tsx

```diff
@@ -1,9 +1,13 @@
-import { Component, ComponentInterface, h, Host } from '@stencil/core'
+import { Component, ComponentInterface, h, Host, Prop } from '@stencil/core'
 
 @Component({
   tag: 'bal-slider-item',
 })
 export class TabSliderItem implements ComponentInterface {
+  /**
+   * Label of the slide which will be used for pagination tabs
+   */
+  @Prop() label?: string
```

Comment:
- [ ] do `@Prop() label = ''` then it is always a string, so you can get rid off the slidesLabels `string | undefined`

Reply:
it was like this but I wanted to avoid other checks in pagination component. I put it back and checked the label value when rendering the tabs